src/cf-layout/src/organisms/featured-content-module.less

```diff
@@ -0,0 +1,87 @@
+
+@fcm-bp-max: 700px;
+@fcm-bp-min: 701px;
+
+.o-featured-content-module {
+  .webfont-regular();
+
+  position: relative;
+
+  min-height: 320px;
+  margin-right: -1px;
+  margin-left: -1px;
+
+  background-color: @block__bg;
+
+  .respond-to-max(@bp-xs-max, {
```

Comment:
This is really minor and I'm looking into a linter to take care of this, but can you space out some variables?
```
.respond-to-max( @bp-xs-max, {
    padding-right: 15px;
    padding-left: 15px;
} );
```
and

`padding-top: unit( @grid_gutter-width / @base-font-size-px, em );`